packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts

```diff
@@ -305,15 +311,25 @@ describe('ThemeStore', () => {
 		expect(baseResponsiveOverrides).toBeDefined();
 
 		// order here matches order merged via theme() getter
-		const merged = mergeThemeLayers(config.base, baseResponsiveOverrides as ThemePartial, config.currency, config.language, config.overrides!);
+		const merged = mergeThemeLayers(
+			config.base,
+			{ components: baseResponsiveOverrides as ThemePartial },
+			config.currency,
+			config.language,
+			config.overrides!
+		);
 
 		expect(store.theme).toStrictEqual({
 			...merged,
 			name: config.name,
+			activeBreakpoint: 'default',
```

Comment:
I'm a bit confused about how 'default' is the active breakpoint when the mobile override '*(M)results' is asserted below.